ansible/roles/webserver-content/tasks/main.yml

```diff
@@ -70,3 +82,33 @@
     block: |
       www-data ALL=(ALL) ALL
       www-data ALL=(ALL) NOPASSWD: /usr/bin/pumount
+
+- name: checkout NextCloud integration
+  git:
+   repo: https://github.com/adavenpo/BibleBox-python-app.git
+   dest: "/tmp/BibleBox-nextcloud"
+   depth: 1
+   #force: yes
+
+- name: checkout latest owncloud lib
+  git:
+    repo: https://github.com/owncloud/pyocclient.git
+    dest: /tmp/pyocclient
+    depth: 1
+    #force=yes
+
+- name: Pull python packages
+  shell: "pip install webapp2 webob"
```

Comment:
Please use the [pip task](http://docs.ansible.com/ansible/pip_module.html). Pretty sure that that the python pip package is not installed by default (not sure what's it's called in debian), so you'll need to add that too using apt.